Compile failed, got  "ld: 31 duplicate symbols for architecture x86_64"

I tried to build solidity from source code on my MacOS following those steps:
```
$ cd $SOURCE_DIR
$ mkdir build && cd build && cmake         # all successed
$ make               #output as below
....
[100%] Linking CXX executable soltest
duplicate symbol __ZNK5boost9unit_test9framework17context_generator8is_emptyEv in:
    CMakeFiles/soltest.dir/boostTest.cpp.o
    /usr/local/lib/libboost_unit_test_framework-mt.a(framework.o)
duplicate symbol __ZN5boost9unit_test9framework11get_contextEv in:
    CMakeFiles/soltest.dir/boostTest.cpp.o
    /usr/local/lib/libboost_unit_test_framework-mt.a(framework.o)
duplicate symbol __ZNK5boost9unit_test9framework17context_generator4nextEv in:
    CMakeFiles/soltest.dir/boostTest.cpp.o
    /usr/local/lib/libboost_unit_test_framework-mt.a(framework.o)
duplicate symbol __ZN5boost9unit_test9framework17current_test_unitEv in:
    CMakeFiles/soltest.dir/boostTest.cpp.o
    /usr/local/lib/libboost_unit_test_framework-mt.a(framework.o)
duplicate symbol __ZN5boost9unit_test9framework16test_in_progressEv in:
    CMakeFiles/soltest.dir/boostTest.cpp.o
    /usr/local/lib/libboost_unit_test_framework-mt.a(framework.o)
duplicate symbol __ZN5boost9unit_test9framework4impl13setup_loggersEv in:
    CMakeFiles/soltest.dir/boostTest.cpp.o
    /usr/local/lib/libboost_unit_test_framework-mt.a(framework.o)
duplicate symbol __ZN5boost9unit_test9framework5clearEv in:
    CMakeFiles/soltest.dir/boostTest.cpp.o
    /usr/local/lib/libboost_unit_test_framework-mt.a(framework.o)
duplicate symbol __ZN5boost9unit_test9framework8shutdownEv in:
    CMakeFiles/soltest.dir/boostTest.cpp.o
    /usr/local/lib/libboost_unit_test_framework-mt.a(framework.o)
duplicate symbol __ZN5boost9unit_test9framework17master_test_suiteEv in:
    CMakeFiles/soltest.dir/boostTest.cpp.o
    /usr/local/lib/libboost_unit_test_framework-mt.a(framework.o)
duplicate symbol __ZN5boost9unit_test9framework17current_test_caseEv in:
    CMakeFiles/soltest.dir/boostTest.cpp.o
    /usr/local/lib/libboost_unit_test_framework-mt.a(framework.o)
duplicate symbol __ZN5boost9unit_test9framework20current_test_case_idEv in:
    CMakeFiles/soltest.dir/boostTest.cpp.o
    /usr/local/lib/libboost_unit_test_framework-mt.a(framework.o)
duplicate symbol __ZN5boost9unit_test9framework12test_abortedEv in:
    CMakeFiles/soltest.dir/boostTest.cpp.o
    /usr/local/lib/libboost_unit_test_framework-mt.a(framework.o)
duplicate symbol __ZN5boost9unit_test9framework20finalize_setup_phaseEm in:
    CMakeFiles/soltest.dir/boostTest.cpp.o
    /usr/local/lib/libboost_unit_test_framework-mt.a(framework.o)
duplicate symbol __ZN5boost9unit_test9framework13clear_contextEi in:
    CMakeFiles/soltest.dir/boostTest.cpp.o
    /usr/local/lib/libboost_unit_test_framework-mt.a(framework.o)
duplicate symbol __ZN5boost9unit_test9framework3runEmb in:
    CMakeFiles/soltest.dir/boostTest.cpp.o
    /usr/local/lib/libboost_unit_test_framework-mt.a(framework.o)
duplicate symbol __ZN5boost9unit_test9framework3runEPKNS0_9test_unitEb in:
    CMakeFiles/soltest.dir/boostTest.cpp.o
    /usr/local/lib/libboost_unit_test_framework-mt.a(framework.o)
duplicate symbol __ZN5boost9unit_test9framework11add_contextERKNS0_12lazy_ostreamEb in:
    CMakeFiles/soltest.dir/boostTest.cpp.o
    /usr/local/lib/libboost_unit_test_framework-mt.a(framework.o)
duplicate symbol __ZN5boost9unit_test9framework23current_auto_test_suiteEPNS0_10test_suiteEb in:
    CMakeFiles/soltest.dir/boostTest.cpp.o
    /usr/local/lib/libboost_unit_test_framework-mt.a(framework.o)
duplicate symbol __ZN5boost9unit_test9framework4initEPFPNS0_10test_suiteEiPPcEiS5_ in:
    CMakeFiles/soltest.dir/boostTest.cpp.o
    /usr/local/lib/libboost_unit_test_framework-mt.a(framework.o)
duplicate symbol __ZN5boost9unit_test9framework16assertion_resultENS0_16assertion_resultE in:
    CMakeFiles/soltest.dir/boostTest.cpp.o
    /usr/local/lib/libboost_unit_test_framework-mt.a(framework.o)
duplicate symbol __ZN5boost9unit_test9framework20deregister_test_unitEPNS0_9test_unitE in:
    CMakeFiles/soltest.dir/boostTest.cpp.o
    /usr/local/lib/libboost_unit_test_framework-mt.a(framework.o)
duplicate symbol __ZN5boost9unit_test9framework4impl19setup_for_executionERKNS0_9test_unitE in:
    CMakeFiles/soltest.dir/boostTest.cpp.o
    /usr/local/lib/libboost_unit_test_framework-mt.a(framework.o)
duplicate symbol __ZN5boost9unit_test9framework17test_unit_abortedERKNS0_9test_unitE in:
    CMakeFiles/soltest.dir/boostTest.cpp.o
    /usr/local/lib/libboost_unit_test_framework-mt.a(framework.o)
duplicate symbol __ZN5boost9unit_test9framework19deregister_observerERNS0_13test_observerE in:
    CMakeFiles/soltest.dir/boostTest.cpp.o
    /usr/local/lib/libboost_unit_test_framework-mt.a(framework.o)
duplicate symbol __ZN5boost9unit_test9framework17register_observerERNS0_13test_observerE in:
    CMakeFiles/soltest.dir/boostTest.cpp.o
    /usr/local/lib/libboost_unit_test_framework-mt.a(framework.o)
duplicate symbol __ZN5boost9unit_test9framework16exception_caughtERKNS_19execution_exceptionE in:
    CMakeFiles/soltest.dir/boostTest.cpp.o
    /usr/local/lib/libboost_unit_test_framework-mt.a(framework.o)
duplicate symbol __ZN5boost9unit_test9framework18register_test_unitEPNS0_10test_suiteE in:
    CMakeFiles/soltest.dir/boostTest.cpp.o
    /usr/local/lib/libboost_unit_test_framework-mt.a(framework.o)
duplicate symbol __ZN5boost9unit_test9framework18register_test_unitEPNS0_9test_caseE in:
    CMakeFiles/soltest.dir/boostTest.cpp.o
    /usr/local/lib/libboost_unit_test_framework-mt.a(framework.o)
duplicate symbol __ZN5boost9unit_test9framework25deregister_global_fixtureERNS0_17test_unit_fixtureE in:
    CMakeFiles/soltest.dir/boostTest.cpp.o
    /usr/local/lib/libboost_unit_test_framework-mt.a(framework.o)
duplicate symbol __ZN5boost9unit_test9framework23register_global_fixtureERNS0_17test_unit_fixtureE in:
    CMakeFiles/soltest.dir/boostTest.cpp.o
    /usr/local/lib/libboost_unit_test_framework-mt.a(framework.o)
duplicate symbol __ZN5boost9unit_test9framework3getEmNS0_14test_unit_typeE in:
    CMakeFiles/soltest.dir/boostTest.cpp.o
    /usr/local/lib/libboost_unit_test_framework-mt.a(framework.o)
ld: 31 duplicate symbols for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [test/soltest] Error 1
make[1]: *** [test/CMakeFiles/soltest.dir/all] Error 2
make: *** [all] Error 2
```
Any one met the same problem?

--- a/test/boostTest.cpp
+++ b/test/boostTest.cpp
@@ -28,7 +28,7 @@
 #pragma warning(push)
 #pragma warning(disable:4535) // calling _set_se_translator requires /EHa
 #endif
-#include <boost/test/included/unit_test.hpp>
+#include <boost/test/unit_test.hpp>
 #if defined(_MSC_VER)
 #pragma warning(pop)
 #endif

